src/land/smap_ssm2ioda.py

```diff
@@ -85,6 +86,9 @@ def _read(self):
         lons = ncd.groups['Soil_Moisture_Retrieval_Data'].variables['longitude'][:].ravel()
         errs = ncd.groups['Soil_Moisture_Retrieval_Data'].variables['soil_moisture_error'][:].ravel()
         qflg = ncd.groups['Soil_Moisture_Retrieval_Data'].variables['retrieval_qual_flag'][:].ravel()
+
+        deps = 0.0*vals
+        deps[:] = 0.025
```

Comment:
Same comment as above. Eliminate unnecessary operations:
```suggestion
        deps = np.full_like(vals, 0.025)
```

Reply:
Revised as you suggested. Thank you @srherbener